bears/js/JSHintBear.py

```diff
@@ -21,407 +18,357 @@ def bool_or_int(value):
         return int(value)
 
 
-class JSHintBear(LocalBear, Lint):
-    executable = 'jshint'
-    output_regex = re.compile(
-        r'.+?: line (?P<line>\d+), col (?P<col>\d+), '
-        r'(?P<message>.+) \((?P<severity>\S)\d+\)')
-    severity_map = {
-        "E": RESULT_SEVERITY.MAJOR,
-        "W": RESULT_SEVERITY.NORMAL,
-        "I": RESULT_SEVERITY.NORMAL}
+@linter(executable='jshint',
+        output_format='regex',
+        output_regex=r'.+?: line (?P<line>\d+), col (?P<column>\d+), '
+                     r'(?P<message>.+) \((?P<severity>[EWI])\d+\)',
+        severity_map={'E': RESULT_SEVERITY.MAJOR,
+                      'W': RESULT_SEVERITY.NORMAL,
+                      'I': RESULT_SEVERITY.INFO})
+class JSHintBear:
+    """
+    Checks the code with jshint. This will run jshint over each file
+    separately.
+    """
```

Comment:
Compare with

> Check CoffeeScript code for a clean and consistent style.
> 
> For more information about coffeelint, visit <http://www.coffeelint.org/>.

Perhaps unify this sort of descriptions?

I suggest something along these lines:


> Detect errors and potential problems in JavaScript code.
> 
> For more information about jshint, visit <http://jshint.com/>.

This also gives an indication of what the bear does: linting vs style checking (cfr. CoffeeLint).